Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -244,41 +404,78 @@ CDMInstanceOpenCDM::Session::Session(const media::OpenCdm& source, Ref<WebCore::
 
     m_message = SharedBuffer::create(message.c_str() + offset, message.size() - offset);
     m_needsIndividualization = requestType.length() == 1 && static_cast<WebCore::MediaKeyMessageType>(requestType.toInt()) == CDMInstance::MessageType::IndividualizationRequest;
+    for (const auto& challengeCallback : m_challengeCallbacks)
+        challengeCallback(this);
+    m_challengeCallbacks.clear();
+}
+
+void CDMInstanceOpenCDM::Session::didUpdateKey(const std::string& status)
+{
+    for (auto& keyStatusData : m_keyStatusDatas) {
+        m_lastStatus = opencdm_session_status(m_session.get(), keyStatusData.m_key.get(), keyStatusData.m_keySize);
+        keyStatusData.m_callback(this, m_lastStatus, String(status.c_str(), status.empty() ? 0 : status.size()));
+    }
+    m_keyStatusDatas.clear();
+
+    for (auto& loadCallback : m_loadCallbacks) {
+        loadCallback(this, true, String(status.c_str(), status.empty() ? 0 : status.size()));
+    }
```

Comment:
It's not there. This code is no longer there

Reply:
I fixed it after you mentioning it for the first time. Now it's not even there because loadCallback got merged with removeCallback and is now sessionChangedCallback. Anyway. There's no one line conditional or loop surrounded by { }  in the code

Reply:
Ach. Now I also see what this applied to.